Dockerfile-UI

```diff
@@ -5,5 +5,15 @@ RUN npm install && npm run build
 
 FROM nginx:stable-alpine as ui-production-stage
```

Comment:
Alternatively we could just use a base image meant for OpenShift, i.e. https://github.com/sclorg/nginx-container

I leave the decision up to you, but I'd rather use the sclorg base image.